reference/tax_settings.v3.yml

```diff
@@ -170,6 +170,9 @@ components:
             - FIXED
             - BASIC
             - DISABLE
+        should_subtract_store_tax:
```

Comment:
😅 Just to make doubly sure, since I know there has been confusion here recently, the default is `true` (that is, this is the current state for all stores).

Reply:
Got it, thanks, added the default value!